components/script_traits/script_msg.rs

```diff
@@ -2,11 +2,12 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+use ::IFrameLoadInfo;
```

Comment:
This `::` shouldn't be necessary. Be sure to run `./mach test-tidy` after removing it, because the expected sort ordering will change.